src/gui/credits.rs

```diff
@@ -0,0 +1,178 @@
+    }
+}
+
+fn create_game_credits() -> Vec<CreditLabel> {
+    let list: [(String, LabelType); 31] = [
+        ("Fish Fight".to_string(), LabelType::MainHeader),
+        ("Founding Team".to_string(), LabelType::SecondaryHeader),
+        (
+            "Erlend Sogge Heggen - General Director".to_string(),
+            LabelType::Regular,
+        ),
+        (
+            "Jacob Terjesønn Rypdal- Sound Director".to_string(),
+            LabelType::Regular,
+        ),
+        (
+            "Fedor Logachev - Technical Director".to_string(),
+            LabelType::Regular,
+        ),
+        (
+            "Ole A. Sjo Fasting - Technical Director".to_string(),
+            LabelType::Regular,
+        ),
+        (
+            "Orlando Herrera - Art Director".to_string(),
+            LabelType::Regular,
+        ),
+        ("Regulars".to_string(), LabelType::SecondaryHeader),
+        (
+            "Orhun Parmaksiz - Infrastructure".to_string(),
+            LabelType::Regular,
+        ),
+        (
+            "Kultaev Aleksandr - Gameplay Code".to_string(),
+            LabelType::Regular,
+        ),
+        (
+            "Carlo Supina - Dev Advocacy".to_string(),
+            LabelType::Regular,
+        ),
+        (
+            "Micah Tigley - Dev Advocacy".to_string(),
+            LabelType::Regular,
+        ),
+        ("Drake - Media Content".to_string(), LabelType::Regular),
+        ("Zyrafa - Gameplay Design".to_string(), LabelType::Regular),
+        ("Contributors".to_string(), LabelType::SecondaryHeader),
+        (
+            "Saverio Miroddi - Gameplay Code".to_string(),
+            LabelType::Regular,
+        ),
+        ("@grufkork - Gameplay Code".to_string(), LabelType::Regular),
+        (
+            "@PotatoTechgit - Gameplay Code".to_string(),
+            LabelType::Regular,
+        ),
+        (
+            "Armando Gonzalez - Gameplay Code".to_string(),
+            LabelType::Regular,
+        ),
+        ("Isaac - Gameplay Code".to_string(), LabelType::Regular),
+        (
+            "Kadir Yazici - Gameplay Code".to_string(),
+            LabelType::Regular,
+        ),
+        (
+            "Ignat Insarovi - Gameplay Code".to_string(),
+            LabelType::Regular,
+        ),
+        ("garoslaw - Audio".to_string(), LabelType::Regular),
+        (
+            "Animesh Sahu - Infrastructure".to_string(),
+            LabelType::Regular,
+        ),
+        ("Srayan Jana - Dev Advocacy".to_string(), LabelType::Regular),
+        ("Alve Larsson - Website".to_string(), LabelType::Regular),
+        (
+            "William Batista - Gameplay Code".to_string(),
+            LabelType::Regular,
+        ),
+        ("Tech Foundations".to_string(), LabelType::SecondaryHeader),
+        ("macroquad.rs".to_string(), LabelType::Regular),
+        ("rust-lang.org".to_string(), LabelType::Regular),
+        ("Thank you!".to_string(), LabelType::MainHeader),
+    ];
```

Comment:
I think it would be better if we store this list in a constant variable. Like so:

```rs
const CREDITS_LIST: [(&str, LabelType); 31] = [
    ("Fish Fight", LabelType::MainHeader),
    ("Founding Team", LabelType::SecondaryHeader),
    ("Erlend Sogge Heggen - General Director", LabelType::Regular),
    ("Jacob Terjesønn Rypdal- Sound Director", LabelType::Regular),
    ("Fedor Logachev - Technical Director", LabelType::Regular),
    ("Ole A. Sjo Fasting - Technical Director", LabelType::Regular),
    ("Orlando Herrera - Art Director", LabelType::Regular),
    ("Regulars", LabelType::SecondaryHeader),
    ("Orhun Parmaksız - Infrastructure", LabelType::Regular),
    ("Kultaev Aleksandr - Gameplay Code", LabelType::Regular),
    ("Carlo Supina - Dev Advocacy", LabelType::Regular),
    ("Micah Tigley - Dev Advocacy", LabelType::Regular),
    ("Drake - Media Content", LabelType::Regular),
    ("Zyrafa - Gameplay Design", LabelType::Regular),
    ("Contributors", LabelType::SecondaryHeader),
    ("Saverio Miroddi - Gameplay Code", LabelType::Regular),
    ("@grufkork - Gameplay Code", LabelType::Regular),
    ("@PotatoTechgit - Gameplay Code", LabelType::Regular),
    ("Armando Gonzalez - Gameplay Code", LabelType::Regular),
    ("Isaac - Gameplay Code", LabelType::Regular),
    ("Kadir Yazıcı - Gameplay Code", LabelType::Regular),
    ("Ignat Insarovi - Gameplay Code", LabelType::Regular),
    ("garoslaw - Audio", LabelType::Regular),
    ("Animesh Sahu - Infrastructure", LabelType::Regular),
    ("Srayan Jana - Dev Advocacy", LabelType::Regular),
    ("Alve Larsson - Website", LabelType::Regular),
    ("William Batista - Gameplay Code", LabelType::Regular),
    ("Tech Foundations", LabelType::SecondaryHeader),
    ("macroquad.rs", LabelType::Regular),
    ("rust-lang.org", LabelType::Regular),
    ("Thank you!", LabelType::MainHeader),
];
```

Reply:
That makes sense, done!